app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/DefaultSettingsItem.kt

```diff
@@ -176,27 +176,31 @@ fun DefaultSettingsItem(
                 )
             }
         },
-        warningView = warnings?.let {
-            {
-                Column(
-                    modifier = Modifier.fillMaxWidth(),
-                    verticalArrangement = Arrangement.spacedBy(RadixTheme.dimensions.paddingMedium)
-                ) {
-                    it.forEach { warning ->
-                        Row(
-                            verticalAlignment = CenterVertically,
-                            horizontalArrangement = Arrangement.spacedBy(RadixTheme.dimensions.paddingXSmall)
-                        ) {
-                            Icon(
-                                painter = painterResource(id = R.drawable.ic_warning_error),
-                                contentDescription = null,
-                                tint = RadixTheme.colors.orange3
-                            )
-                            Text(
-                                text = warning,
-                                style = RadixTheme.typography.body2HighImportance,
-                                color = RadixTheme.colors.orange3
-                            )
+        warningView = if (warnings?.isEmpty() == true) {
```

Comment:
can be replaced with `warnings.isNullOrEmpty()` this way avoiding an extra nullability check within `else` block